cmd/quickstart.go

```diff
@@ -52,13 +54,18 @@ func quickstartExec(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	outDir, err := cmd.Flags().GetString("out-dir")
```

Comment:
maybe working dir or something? A bit confusing with the `out` flag on generate

Reply:
The reason I'm not sure if that works is that the working director I would think is where I am executing `speakeasy quickstart` that is not necessarily the same as the output dir.